bin/fabric-service-start.sh

```diff
@@ -20,6 +20,7 @@ serviceID=`docker run -d \
  -e "CERTIFICATION_DURATION=${CERTIFICATION_DURATION}" \
  -e "SERVICE_BIND_PORT=${SERVICE_BIND_PORT}" \
  -e "SERVICE_EXTERNAL_ADDRESS=${SERVICE_EXTERNAL_ADDRESS}" \
+ -e "ADMIN_PWD=${ADMIN_PWD}" \
```

Comment:
lets use PASSWORD to not confuse with anything else

Reply:
renamed.